pusher.gemspec

```diff
@@ -3,7 +3,7 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = "pusher"
-  s.version     = "0.16.0"
+  s.version     = "0.17.0"
```

Comment:
Making a release should be done independently from the PR.

The best way to do a release is as follows:

* bump the gemspec version
* run `bundle` if the Gemfile.lock is part of the repo
* run `git changelog` from the `git-extras` package to generate the changelog, edit for humans
* `git commit -a -m "Release v$version"`
* `git tag v$version; git push; git push --tags`
* `rm *.gem; gem build *.gemspec && gem push *.gem`